src/formatter.ts

```diff
@@ -10,19 +11,21 @@ import {
   writeTodosSync,
   _buildTodoDatum,
 } from '@ember-template-lint/todo-utils';
-import type { ESLint, Linter } from 'eslint';
+
+import hasFlag from 'has-flag';
+// import { getLintTodoConfig } from '@ember-template-lint/todo-utils';
```

Comment:
Remove commented line.

Reply:
Feel free to submit these as suggestions in the future!

Reply:
Done.